packages/manager/src/features/Databases/DatabaseDetail/DatabaseSummary/DatabaseSummaryConnectionDetails.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+// import useDatabases from 'src/hooks/useDatabases';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface Props {
+  // databaseID: number;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const DatabaseSummaryConnectionDetails: React.FC<Props> = (props) => {
+  // const databases = useDatabases();
+  // const { databaseID } = props;
+  // const thisDatabase = databases.databases.itemsById[databaseID];
+
+  return (
+    <>
+      Database Connection Details
+    </>
+  );
```

Comment:
To match spec heading

```suggestion
  return <>Connection Details</>;
```